build/cache/backends/inline.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: "Inline cache"
 keywords: build, buildx, cache, backend, inline
+redirect_from:
+  - /build/building/cache/backends/inline
```

Comment:
@crazy-max do you know if the redirects should be specified with a trailing `/`? (Mostly wondering if there would be double redirects); not at my computer right now to double check 😅

Reply:
Ah yes we need the trailing slash indeed!

Reply:
We should look at a linter for that 🤔 (I _suspect_ both may work, but being consistent of course never hurts)